components/clientComponents/forms/Review/QuestionsAnswersList.tsx

```diff
@@ -0,0 +1,28 @@
+import { randomId } from "@lib/client/clientHelpers";
+import { SubElements } from "./SubElements";
+import { ReviewElement, ReviewItem } from "./reviewUtils";
+
+export const QuestionsAnswersList = ({
+  reviewItem,
+}: {
+  reviewItem: ReviewItem;
+}): React.ReactElement => {
+  return (
+    <dl className="my-10">
+      {Array.isArray(reviewItem.elements) &&
```

Comment:
We should early return a null above the `dl` here if we're not working with an array ---  doesn't make sense to me to output an empty `<dl`